### DISCUSSION TITLE
Why I can't submit the prompt by <C-CR>?

### DISCUSSION THREAD

**Discussion by basheewang** [jackMort/ChatGPT.nvim#158] [Q&A]
I am press <C-CR> in both input and normal mode, but nothing happened... I am also trying to setup the shortcut to <C-m>, but I don't know why it doesn't work. Can anyone help me?

Here is my config, I am using NvChad:
``` lua 
  ["MunifTanjim/nui.nvim"] = { module = "nui" },

  ["jackMort/ChatGPT.nvim"] = {
    cmd = { "ChatGPT", "ChatGPTActAs" },
    config = function()
      require("chatgpt").setup {
        popup_input = {
          submit = "<C-t>",
        },
      }
    end,
  },
```

**Comment by ariel-frischer** [comment]
Your terminal probably send some different keystroke depending on your terminal/pc.
Also, the mapping you have listed `<c-t>` is `ctrl + t` not `ctrl + enter`.  Maybe you want to try `<c-cr>`.
https://stackoverflow.com/a/23549950/1715153
